### ISSUE TITLE
Add "Mixed support" in the Estimated Support count

### ISSUE THREAD

**Issue by hteumeuleu** [hteumeuleu/caniemail#135]
As reported to me by @M-J-Robbins : 

> It looks like the “Estimated support” is including unsupported as partial support if it has a note added. I’d say `aria-labelledby` should be 52.54% rather than 87.21%. https://www.caniemail.com/features/html-aria-labelledby/

I think it might be the way I'm reporting mixed support. For example, for Gmail, the desktop webmail has no support but the mobile clients do. Because there's no way to differentiate those in Litmus stats, I'm counting them all as "partially supported". But I see why this get confusing. I should have both a "partial support" and "mixed support" section in this.

This happens in https://github.com/hteumeuleu/caniemail/blob/master/_includes/feature-support.html

**Comment by hteumeuleu** [comment]
Should be fixed now: https://github.com/hteumeuleu/caniemail/commit/9bddff54bdd006725312d8a98823a51073993673
